src/components/ArticleLabel/index.tsx

```diff
@@ -0,0 +1,44 @@
+import {SFC} from 'types';
+import * as S from './Styles';
+import {Article} from 'types/articles';
+
+export interface ArticleDetailsProps {
+  article: Article;
+  element: any;
+}
+
+const ArticleDetail: SFC<ArticleDetailsProps> = ({className, article, element}) => {
+  const paragraphs = article.detail.split('\n').filter((paragraph) => paragraph.trim() !== '');
```

Comment:
I don't think we should do this.
We should render the content as it is and have new lines addd via CSS